tajo-core/src/main/java/org/apache/tajo/worker/TajoWorker.java

```diff
@@ -200,6 +200,7 @@ public void serviceInit(Configuration conf) throws Exception {
     historyReader = new HistoryReader(workerContext.getWorkerName(), this.systemConf);
 
     FunctionLoader.loadUserDefinedFunctions(systemConf);
+    HiveFunctionLoader.loadHiveUDFs(systemConf);
```

Comment:
If any duplicate exists, it is already designed to raise AmbiguousFunctionException.
So this point might be skipped.